Spell doc 2024-10-17 (#2766)
makerdao/community: governance/votes/Executive vote - October 17, 2024.md

date: 2024-10-17T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] Lockstake Engine Initialization, RWA Vault Debt Ceiling Housekeeping, Pinwheel DAO Resolution, Aave Revenue Share Payment, Spark Proxy Spell - October 17, 2024

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
- The Lockstake Engine will be initialized. This requires a number of subactions:
  - Activate a new MKROSM.
  - Activate a new VoteDelegateFactory contract.
  - Initialize the Lockstake Engine farm in the Flapper.
  - Update Smart Burn Engine parameters.
  - Initialize the Lockstake Engine and borrowing facility.
  - The Airdrop Multisig for Early Bird Rewards will be funded with newly minted SKY tokens.

> - Debt Ceilings for the now defunct Monetalis vaults will be set to 0.

Reviewer: ```suggestion
- Debt Ceilings for the now defunct RWA vaults operated by Monetalis will be set to 0.
```

  - `line`: **18.8 million DAI**.
  - `gap`: **5 million DAI**.
  - `ttl`: **16 hours**.
- LSE-MKR-A will be added to the ilk registry with the provided `name` and `symbol` and the new ilk class of **7**.
- MCD_SPOT, CLIPPER_MOM, OSM_MOM, MCD_END will use the new MKROSM (PIP_MKR) to assess the collateral price.
- PIP_MKR will be added to the OSM_MOM.
- Lockstake Clipper will be configured with the following parameters:
  - `buf`: **1.20**.
  - `tail`: **6,000 seconds**.
  - `cusp`: **0.40**.
  - `chip`: **0.1%**.
  - `tip`: **300 DAI**.
  - `stopped`: **0**.
  - `tolerance`: **0.5**.
- StairstepExponentialDecrease calc contract will be configured with the following parameters:
  - `cut`: **0.99**.
  - `step`: **60 seconds**.
- REWARDS_LSMKR_USDS will be added to the Lockstake Engine as the first farm.
- The LockstakeEngine will be authorized to acccess the `vat`.
- The LockstakeClipper will be authorized to access the `vat` and the LockstakeEngine.
- CLIPPER_MOM, MCD_DOG and MCD_END will be authorized to access LockstakeClipper.
- New chainlog keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP and LOCKSTAKE_CLIP_CALC will be added.

#### Airdrop Multisig Funding

> To facilitate the payment of [Early Bird Rewards](https://sky-atlas.powerhouse.io/#A.5.2.1.1_Reward_System-99e76538-f850-4937-8092-d166cae5aff4|8d5a9e88cf49) **28,220,925.60 SKY** will be minted to a multisig controlled by the Accessibility Facilitators at [0x14D98650d46BF7679BBD05D4f615A1547C87Bf68](https://etherscan.io/address/0x14D98650d46BF7679BBD05D4f615A1547C87Bf68).

Reviewer: ```suggestion
To facilitate the payment of [Early Bird Rewards](https://sky-atlas.powerhouse.io/#A.5.2.1.1_Reward_System-99e76538-f850-4937-8092-d166cae5aff4|8d5a9e88cf49) **27,222,832.80 SKY** will be minted to a multisig controlled by the Accessibility Facilitators at [0x14D98650d46BF7679BBD05D4f615A1547C87Bf68](https://etherscan.io/address/0x14D98650d46BF7679BBD05D4f615A1547C87Bf68).
```

---
title: Template - [Executive Vote] Seal Engine Initialization, RWA Vault Debt Ceiling Housekeeping, Pinwheel DAO Resolution, Aave-SparkLend Revenue Share Payment for Q3 2024, Spark Proxy Spell - October 17, 2024

> summary: Initialize the Seal Engine, reduce debt ceilings of deprecated RWA vaults, approve DAO Resolution for Pinwheel, transfer payment for the Aave-SparkLend Revenue Share for Q3 2024, execute a Spark Proxy Spell.

Reviewer 2: ```suggestion
summary: Initialize the Seal Engine, fund the Multisig responsible for distributing Early Bird Rewards, reduce debt ceilings of deprecated RWA vaults, approve DAO Resolution for Pinwheel, transfer payment for the Aave-SparkLend Revenue Share for Q3 2024, execute a Spark Proxy Spell.
```
I would also add this here

Reviewer: Good catch.

date: 2024-10-17T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] Seal Engine Initialization, RWA Vault Debt Ceiling Housekeeping, Pinwheel DAO Resolution, Aave Revenue Share Payment, Spark Proxy Spell - October 17, 2024

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

> - The Seal Engine will be initialized. This requires a number of subactions:

Reviewer 3: ```suggestion
- The Seal Engine will be initialized. This requires a number of sub-actions:
```

  - Activate a new MKROSM.
  - Activate a new VoteDelegateFactory contract.
  - Initialize the Seal Engine farm in the Flapper.
  - Update Smart Burn Engine parameters.
  - Initialize the Seal Engine and borrowing facility.

>   - The Multisig responsible for distributing Early Bird Rewards will be funded with newly minted SKY tokens.

Reviewer 3: ```suggestion
  - Fund the Multisig responsible for distributing Early Bird Rewards with newly minted SKY tokens.
```
For the sake of consistency and ease of reading

- A Spark proxy spell at [0xcc3B9e79261A7064A0f734Cc749A8e3762e0a187](https://etherscan.io/address/0xcc3B9e79261A7064A0f734Cc749A8e3762e0a187) will be executed.

**Voting for this executive proposal will place your MKR/SKY in support of the actions outlined above.**

Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Sky Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**16 hours**](https://sky-atlas.powerhouse.io/#A.1.8.2.1.2_Pause_Delay_Current_Value-09d2514b-3169-4755-a654-2c774456980d|0db30758e055d2d0).

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.

---

## Proposal Details

### Seal Engine Initialization

> - **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmUm8Krq)

Reviewer 3: @MakerJanSky this is missing in the Exec. Sheet

Reviewer: Fixed. Thank you, CivicSage.

- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-weekly-cycle-edit-proposal-week-of-october-14-2024-01/25324)

> If this executive proposal passes, then the Seal Engine will be initialized through the following subactions.

Reviewer 3: ```suggestion
If this executive proposal passes, then the Seal Engine will be initialized through the following sub-actions:
```

#### MKROSM Activation

> - The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will be authorized to read the MKR oracle price from the current PIP_MKR at [0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df](https://etherscan.io/address/0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df).

Reviewer 2: ```suggestion
- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will be whitelisted to read the MKR oracle price from the current PIP_MKR at [0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df](https://etherscan.io/address/0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df).
```
Based on Dewiz’s feedback, shouldn’t we be using the verb "whitelist" instead of "authorize" in these cases?

- The new MKROSM at [0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b](https://etherscan.io/address/0x4F94e33D0D74CfF5Ca0D3a66F1A650628551C56b) will replace the current PIP_MKR in the [Chainlog](https://chainlog.sky.money/).

#### New VoteDelegateFactory Contract Activation

> - The current VOTE_DELEGATE_PROXY_FACTORY in the Chainlog](https://chainlog.sky.money/) at [0xd897f108670903d1d6070fcf818f9db3615af272](https://etherscan.io/address/0xd897f108670903d1d6070fcf818f9db3615af272) will be renamed to VOTE_DELEGATE_FACTORY_LEGACY.

Reviewer 3: ```suggestion
- The current VOTE_DELEGATE_PROXY_FACTORY in the [Chainlog](https://chainlog.sky.money/) at [0xd897f108670903d1d6070fcf818f9db3615af272](https://etherscan.io/address/0xd897f108670903d1d6070fcf818f9db3615af272) will be renamed to VOTE_DELEGATE_FACTORY_LEGACY.
```

  - `stopped`: **0**.
  - [`tolerance`](https://sky-atlas.powerhouse.io/#A.1.8.2.2.4.1_Liquidations_Circuit_Breaker_Exception_Price_Tolerance-bcccd6b0-ad61-47c6-94f1-93447ba39102|0db30758e055352ce6eb): **0.5**.
- StairstepExponentialDecrease calc contract will be configured with the following parameters:
  - [`cut`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.1_Auction_Price_Function_(cut)-b6e47d05-addf-4371-ad95-90582da7c0f1|57eaf45219bea3b430c2a59a6013): **0.99**.
  - [`step`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.2_Auction_Price_Function_(step)-7d58c961-43ff-4157-80aa-8a92f577ec12|57eaf45219bea3b430c2a59a6013): **60 seconds**.
- REWARDS_LSMKR_USDS will be added to the Seal Engine as the first farm.
- The LockstakeEngine contract will be authorized to access the `vat`.
- The LockstakeClipper contract will be authorized to access the `vat` and the LockstakeEngine.
- CLIPPER_MOM, MCD_DOG, and MCD_END will be authorized to access LockstakeClipper.
- New [Chainlog](https://chainlog.sky.money) keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP, and LOCKSTAKE_CLIP_CALC will be added.

> #### Early Bird Rewards Multisig Funding

Reviewer 2: ```suggestion
### Early Bird Rewards Multisig Funding
```
Imo, this is not related to the Seal Engine Initialization

Reviewer: Fair assessment. I will change the Executive Summary to reflect this as well.

Once the required SKY tokens have been transferred to the MerkleDistributor it is expected that the remaining tokens will be burned.

### RWA Debt Ceiling Reductions

- **Authorization**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323/2)
- **Proposal**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323)

If this executive proposal passes, then the following Debt Ceiling changes for the now deprecated RWA Vaults operated by Monetalis will be carried out.

#### Monetalis Clydesdale (RWA007-A)

- RWA007-A will be removed from the [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).

> - RWA007-A Debt Ceiling will be set to **0 DAI**.

Reviewer 2: ```suggestion
- Reduce the RWA007-A Debt Ceiling by 3 billion DAI from 3 billion DAI to **0 DAI**.
```
This is more of a question than a suggestion. Why aren’t we using the same format as the Coinbase Custody Debt Ceiling change? Is it because Monetalis Clydesdale uses the DC-IAM?

Reviewer: You are correct. Monetalis is on the DC-IAM while Coinbase Custody is not.

Reviewer: And it has to be removed from the DC-IAM in order to set the DC to 0.

Reviewer: Would you like to resolve this one @VoteWizard , or would you still like it changed? We feel it is correct as is, but don't mind the suggestion if you prefer it.

#### Coinbase Custody (RWA014-A)

RWA014-A does not use the [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).

- Reduce the RWA014-A Debt Ceiling by 1.5 billion DAI from 1.5 billion DAI to **0 DAI**.

#### Oracle Adjustments

As part of the offboarding process, `tell` and `cull` will be called on the oracles for these vaults.

> Calling `tell` will initiate soft liquidation of these vaults and calling `cull` will write off their debt and set the oracle prices to 0. In reality, as neither of these vaults have associated debt the only change will be to set the reported prices of their oracles to 0.

Reviewer 3: ```suggestion
Calling `tell` will initiate soft liquidation of these vaults and calling `cull` will write off their debt and set the oracle prices to 0. In reality, as neither of these vaults have associated debt, the only change will be to set the reported prices of their oracles to 0.
```
